symfony/translation/3.3/etc/packages/translation.yaml

```diff
@@ -0,0 +1,11 @@
+parameters:
+    # Change this locale depending on your app's default language
+    locale: 'en'
+
+framework:
+    default_locale: '%locale%'
+    translator:
+        paths:
+            - '%kernel.project_dir%/etc/translations/'
```

Comment:
I would use `%kernel.project_dir%/translations/` instead as translations as as important as `templates/` for instance when you have a translated project.